samples/wifi/twt/prj.conf

```diff
@@ -90,5 +90,7 @@ CONFIG_NET_CONFIG_MY_IPV4_GW="192.168.1.1"
 CONFIG_NET_MGMT_EVENT_QUEUE_TIMEOUT=5000
 
 CONFIG_TRAFFIC_GEN=y
+# IPv4 address of the host running the traffic generator server
+CONFIG_TRAFFIC_GEN_REMOTE_IPV4_ADDR="192.168.0.1"
```

Comment:
This was intentionally left out of default config, we don't have put some default address without user's knowledge, it will start failing and user has no idea why. So, a build assert is added to force the user to choose an IP, see https://github.com/nrfconnect/sdk-nrf/blob/main/samples/wifi/twt/modules/traffic_gen/src/main.c#L319